src/super_gradients/recipes/dataset_params/cityscapes_segformer_dataset_params.yaml

```diff
@@ -0,0 +1,45 @@
+defaults:
+  - cityscapes_dataset_params
+  - _self_
+
+train_dataset_params:
+  transforms:
+    - SegColorJitter:
+        brightness: 0.5
+        contrast: 0.5
+        saturation: 0.5
+
+    - SegRandomFlip:
+        prob: 0.5
+
+    - SegRandomRescale:
+        scales: [ 0.5, 2.0 ]
+
+    - SegPadShortToCropSize:
+        crop_size: [ 1024, 1024 ]
+        fill_mask: 19
+
+    - SegCropImageAndMask:
+        crop_size: [ 1024, 1024 ]
+        mode: random
+
+val_dataset_params:
+  transforms:
+    - SegRescale:
```

Comment:
If I read this transformation chain correctly, then we
1) Resize input image to ensure smallest size is 1024
2) Pad image to 1024x1024 (Looks it it redundant because we just did resize than guarantee that smallest size is no smaller than 1024)
3) Center crop. This step looks wrong to me as we validate only on central part of the image.

Perhaps, instead we want to do
1) Resize input image to ensure **longest** size is 1024
2) Pad to 1024x1024
 
Please let me know what you think or did I miss anything